app/src/main/java/edu/uc/group/rankine/utilities/DynamicFieldUtil.kt

```diff
@@ -121,6 +121,9 @@ class DynamicFieldUtil(activity: Activity) : Application() {
 
     }
 
+    /**
+     * filters based on the users preferences
```

Comment:
Proper KDoc should also describe parameters, with @param, and return values, with @return.